### PR TITLE
[13.0][MIG] stock_inventory_exclude_sublocation

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Migration to v13 tests simplification as well as Inventory Adjustment options have been modified.

CC @ForgeFlow

### CODE REVIEW

File: stock_inventory_exclude_sublocation/__manifest__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "name": "Stock Inventory Exclude Sublocation",
     "summary": "Allow to perform inventories of a location without including "
     "its child locations.",
-    "version": "12.0.1.0.1",
+    "version": "13.0.1.0.1",
     "development_status": "Mature",
     "author": "Eficent, " "Tecnativa, " "Odoo Community Association (OCA)",
```
Comment (AdriaGForgeFlow) [inline stock_inventory_exclude_sublocation/__manifest__.py]: Could you remove unnecessary quotes, change Eficent to ForgeFlow and also update the Copyright note?

**Review by AdriaGForgeFlow — COMMENTED** [review]
Is there a reason to remove the tests?

Also small comment below

File: stock_inventory_exclude_sublocation/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Stock Inventory Exclude Sublocation",
+    "summary": "Allow to perform inventories of a location without including "
+    "its child locations.",
+    "version": "13.0.1.0.1",
+    "development_status": "Mature",
+    "author": "Eficent, " "Tecnativa, " "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "category": "Warehouse Management",
```
Comment (rousseldenis) [inline stock_inventory_exclude_sublocation/__manifest__.py]: Change this to "Warehouse"

**Review by rousseldenis — APPROVED** [review]
Code review. Minor change

**Comment by HviorForgeFlow** [comment]
> Is there a reason to remove the tests?

Stock Inventory options has been simplify, that tests was testing old features.

@pedrobaeza does it make sense to you?

File: stock_inventory_exclude_sublocation/__manifest__.py (hunk 1 above)
Comment (HviorForgeFlow) [reply stock_inventory_exclude_sublocation/__manifest__.py]: Done!

File: stock_inventory_exclude_sublocation/__manifest__.py (hunk 2 above)
Comment (HviorForgeFlow) [reply stock_inventory_exclude_sublocation/__manifest__.py]: Done!

**Comment by pedrobaeza** [comment]
> does it make sense to you?

Yes if the new flows are covered.

**Review by AdriaGForgeFlow — APPROVED** [review]

**Comment by LoisRForgeFlow** [comment]
Thanks for the work!

/ocabot merge

**Pull request merged**